### PR TITLE
T36372 Change nodes with new schema

### PR DESCRIPTION
**Pull request by JenySadadia**
Depends on https://github.com/kernelci/kernelci-api/pull/155 and https://github.com/kernelci/kernelci-pipeline/pull/62

Add Node.result and change the value of Node.status for the tarball node based on the new Node model schema.

### CODE REVIEW

**Comment by gctucker** [comment]
Let's wait for #62 to be merged before reviewing this one as it depends directly on it.

**Review by gctucker — COMMENTED** [review]
> src/tarball: modify tarball node

> src/runner: modify event filter dict

Please replace the commit subjects to be more descriptive of the changes.

File: src/send_kcidb.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def __call__(self, configs, args):
             while True:
                 event = db.get_event(sub_id)
                 node = db.get_node_from_event(event)
-                if node['name'] != 'checkout' or node['status'] != 'pass':
+                if node['name'] != 'checkout' or node['result'] != 'pass':
```
[the remark this answers is not in the document]
Comment (JenySadadia) [reply src/send_kcidb.py:74]: Honestly, I am not entirely aware of the design of KCIDB. Why are we only sending checkout nodes?

[the remark this answers is not in the document]
Comment (JenySadadia) [reply src/send_kcidb.py:74]: By the way, we still have not implemented any logic to mark the root node 'completed'.

**Review by gctucker — APPROVED** [review]

**Pull request merged**